Scripts/Runtime/anvil-unity-dots-runtime.asmdef

```diff
@@ -9,7 +9,8 @@
         "Unity.Entities",
         "Unity.Jobs",
         "Unity.Mathematics",
-        "Unity.Transforms"
+        "Unity.Transforms",
+        "Unity.Profiling.Core"
```

Comment:
Should this be an editor only reference or conditional on a define?

Reply:
Hmmn, I'm not sure.

It's not an Editor only reference as you will want to profile a build. 
Technically the conditional to include this is `#ENABLE_PROFILER` but I don't see a way to do that. All of their code is commented out if that define isn't in there so I think it's fine to reference it.